mRemoteV1/Themes/ThemeManager.cs

```diff
@@ -104,31 +101,38 @@ public  List<ThemeInfo> LoadThemes()
                         }
                     }
 
-
                     //Load the embedded themes, extended palettes are taken from the vs2015 themes, trying to match the color theme
-                    var vs2003 = new ThemeInfo("Vs2003", new VS2003Theme(), "", VisualStudioToolStripExtender.VsVersion.Vs2003, ((ThemeInfo)themes["vs2015light"]).ExtendedPalette);
```

Comment:
We'll  have to rename at least the 3 vs*.theme files in https://github.com/mRemoteNG/mRemoteNG/tree/35fada7fcd535777899cd00d99763766ecafd717/mRemoteV1/Resources/Themes then....

Those 3 theme names will conflict with the DPS out of the box themes.

Moving/renaming the 3 files will likely cause less problems, but can't really redirect since they share the same name... Not sure which ones are in use by more users though... 

Let me know what you think - I'm fine either way, they are simple changes...

Reply:
Maybe no further change is necessary. I'll do some more testing around this.

Reply:
OK... I think I've got this taken care of in 6156ce48acc1000bc10b783ae04c673f8990d358

I feel this is the best solution and should go largely unnoticed by our users...

Reply:
a few more changes related to themes after feedback from @jcefoli on gitter. I think this is very solid not. Marking this one as resolved.